src/main/resources/application-dev.properties

```diff
@@ -12,6 +12,5 @@ ccd.data.store.api.url=http://localhost:4452
 
 tornado.url=http://localhost:9095/api/render
 case_document_am.url =http://localhost:4455
-feature.secure-doc-store.enabled = ${SECURE_DOC_STORE_FEATURE:false}
+feature.secure-doc-store.enabled = true
```

Comment:
You can just delete this line now as application.properties is 
feature.secure-doc-store.enabled = ${SECURE_DOC_STORE_FEATURE:true}